.eslintignore

```diff
@@ -10,3 +10,6 @@ libs/*
 # not seem to be a reason why we will want to risk being inconsistent with our
 # remaining JavaScript source code.
 !.eslintrc.js
+
+# Webassemblt glue code automatically generated by emscripten
+rnnoise.js
```

Comment:
do we really need this, given we bundle the result (for now) ?

Reply:
Added it because the rnnoise.js doesn't abide by our lint rules and it came up when trying to commit. As we moved this component to the separate rnnoise wasm repository this was removed.